From 7ed54c6155982bd8376fdb88bed79c664d5e111f Mon Sep 17 00:00:00 2001
From: Markos Chandras <hwoarang@gentoo.org>
Date: Thu, 12 Mar 2015 20:52:11 +0000
Subject: [PATCH] templates: gentoo.common: Add /dev/shm tmpfs mount entry

/dev/shm needs to be mounted as tmpfs. It's needed by python
and possibly other packages.

Signed-off-by: Markos Chandras <hwoarang@gentoo.org>
---
Applied in master but not in 1.1.1. It will probably make it
to 1.1.2
---
 config/templates/gentoo.common.conf.in | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/config/templates/gentoo.common.conf.in b/config/templates/gentoo.common.conf.in
index ca3ffc1..49cd411 100644
--- a/config/templates/gentoo.common.conf.in
+++ b/config/templates/gentoo.common.conf.in
@@ -21,3 +21,7 @@ lxc.cgroup.devices.allow = c 10:232 rwm
 ## To use loop devices, copy the following line to the container's
 ## configuration file (uncommented).
 #lxc.cgroup.devices.allow = b 7:* rwm
+
+# /dev/shm needs to be mounted as tmpfs. It's needed by python (bug #496328)
+# and possibly other packages.
+lxc.mount.entry = none dev/shm tmpfs rw,nosuid,nodev,create=dir
-- 
2.3.2

